OpenRA.Game/Graphics/Util.cs

```diff
@@ -248,6 +266,41 @@ public static float[] MatrixMultiply(float[] lhs, float[] rhs)
 			return mtx;
 		}
 
+		/// <summary>
+		/// Rotates a quad from a center point around its Z-axis by a given number of radians
+		/// </summary>
+		/// <param name="center">A float3 containing the center point of the quad</param>
+		/// <param name="size"> A float3 containing the X, Y, and Z lengths of the quad</param>
+		/// <param name="rotation">The number of radians to rotate by</param>
+		/// <returns>An array of four vertices representing the rotated quad (top-left, top-right, bottom-right, bottom-left)</returns>
+		#pragma warning disable SA1025 // Code should not contain multiple whitespace in a row
```

Comment:
This shouldn’t be necessary.

Reply:
Have removed along with the method that it belonged to.